src/fontdata.cpp

```diff
@@ -74,3 +74,34 @@ void FontData::initFromStr(const string& source)
                 string("cannot allocate font \'") + source + "\'");
     }
 }
+
+/**
+ * @brief compute the with of the given text
+ * @param text The text
+ * @param glyphs Only consider that many glyphs from the given text
```

Comment:
the second parameter of this method is "len"

Reply:
I've fixed it, thanks! It seems that depending on the font used, this could be either the number of glyphs or the number of bytes, so I've also updated the description accordingly.

Reply:
Thanks!